modules/imported-account/README.md

```diff
@@ -0,0 +1,87 @@
+# Component: `imported-account`
+
+This component is optional and is used to create the `OrganizationAccountAccessRole` role in an imported account.
```

Comment:
If the description is to simply add a role to a specific account, couldn't the name of this component be something like `iam-role` and allow role assumption override via `profile` / `import_role_arn` / etc and then we can make this component more generic?